analyses/methylation-summary/tools/03-methyl-tpm-correlation.cwl

```diff
@@ -0,0 +1,48 @@
+cwlVersion: v1.2
+class: CommandLineTool
+id: 03-methyl-tpm-correlation
+doc: >-
+  Calculate representative probe-level correlations between rna-seq (tpm) and methylation (beta/m-vlaues) for patients who have samples in both datasets
+
+requirements:
+  - class: ShellCommandRequirement
+  - class: DockerRequirement
+    dockerPull: "pgc-images.sbgenomics.com/d3b-bixu/open-pedcan:latest"
+  - class: InlineJavascriptRequirement
+  - class: ResourceRequirement
+    ramMin: ${return inputs.ram * 1000}
+    coresMin: 4
```

Comment:
why 4 and not 1 ? does this script requires 4 cores?

Reply:
It probably doesn't need 4. When I run it, I'll try with 1.